app/js/validateUserAdditions.js

```diff
@@ -97,19 +98,26 @@ document.querySelector('#addNewUserForm').addEventListener('submit', function(ev
             emailIsValid = false
             errorField.innerHTML = "Your email is not valid or already exists: Please provide a correct email"
         }
-        if (timeField.value <=1 || timeField.value == null || isNaN(timeField.value) === true ) {
+        if (minutesField.value <=1 || minutesField.value == null || isNaN(minutesField.value) === true ) {
+            timeIsValid = false
+            errorField.innerHTML += 'This is not a good number!'
```

Comment:
Could the use textContent rather than innerHTML for these error statements? I see innerHTML has been previously used so not sure if it's within the scope to change it or be consistent

Reply:
As Charlie said, can just them the way they are.

Reply:
```suggestion
            errorField.textContent += 'This is not a good number!'
```